manual/udts/README.md

```diff
@@ -1,5 +1,99 @@
 ## User-defined types
 
-*Coming soon... In the meantime, see the javadoc for [UserType].*
+[CQL user-defined types][cql_doc] are ordered sets of named, typed fields. They must be defined in a
```

Comment:
Should we add a note that UDTs can be frozen / non-frozen?

Reply:
I think in the context of this documentation which is to demonstrate how to use the Driver API, I think it is ok to not cover it here.  Up until a recent C* 3.6 UDTs were required to be frozen, so as long as our schema examples include frozen it should be ok.